services/download/src/test/java/app/coronawarn/server/services/download/FederationBatchRepositoryTest.java

```diff
@@ -0,0 +1,97 @@
+package app.coronawarn.server.services.download;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+import app.coronawarn.server.common.persistence.domain.FederationBatch;
+import app.coronawarn.server.common.persistence.domain.FederationBatchStatus;
+import app.coronawarn.server.common.persistence.repository.FederationBatchRepository;
+import java.time.LocalDate;
+import java.util.Objects;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.reactive.AutoConfigureWebTestClient;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@SpringBootTest
+@ExtendWith({SpringExtension.class})
+@AutoConfigureWebTestClient
+public class FederationBatchRepositoryTest {
```

Comment:
Shouldn't this test be in the persistence module? Since the class `FederationBatchRepository` is there.